Library/Homebrew/extend/os/mac/cmd/config.rb

```diff
@@ -0,0 +1,191 @@
+require "hardware"
```

Comment:
What is the purpose of this file? I was of the impression that the OS-X-specific code of `cmd/config.rb` is now in `Library/Homebrew/extend/os/mac/system_config.rb`.